eegnb/experiments/visual_ssvep/ssvep.py

```diff
@@ -36,28 +22,7 @@ def present(duration=120, eeg=None, save_fn=None):
 
     # Generate the possible ssvep frequencies based on monitor refresh rate
     def get_possible_ssvep_freqs(frame_rate, stim_type="single"):
-        """Get possible SSVEP stimulation frequencies.
-        Utility function that returns the possible SSVEP stimulation
-        frequencies and on/off pattern based on screen refresh rate.
-        Args:
-            frame_rate (float): screen frame rate, in Hz
-        Keyword Args:
-            stim_type (str): type of stimulation
-                'single'-> single graphic stimulus (the displayed object
-                    appears and disappears in the background.)
-                'reversal' -> pattern reversal stimulus (the displayed object
-                    appears and is replaced by its opposite.)
-        Returns:
-            (dict): keys are stimulation frequencies (in Hz), and values are
-                lists of tuples, where each tuple is the number of (on, off)
-                periods of one stimulation cycle
-        For more info on stimulation patterns, see Section 2 of:
-            Danhua Zhu, Jordi Bieger, Gary Garcia Molina, and Ronald M. Aarts,
-            "A Survey of Stimulation Methods Used in SSVEP-Based BCIs,"
-            Computational Intelligence and Neuroscience, vol. 2010, 12 pages,
-            2010.
-        """
-
```

Comment:
Why remove this?

Reply:
Ignore the visual_ssvep for now, still doing it. But I'm gonna keep that in the load_stimulus or present_stimulus function